.swiftlint.yml

```diff
@@ -41,10 +41,6 @@ excluded:
     - "wire-ios-protos/Protos/messages.pb.swift"
     - "wire-ios-protos/Protos/mls.pb.swift"
     - "wire-ios-protos/Protos/otr.pb.swift"
-    - "wire-ios-data-model/sourcery/generated/AutoMockable.generated.swift"
-    - "wire-ios-sync-engine/sourcery/generated/AutoMockable.generated.swift"
-    - "wire-ios-sync-engine/sourcery/generated/AutoMockable.manual.swift"
-    - "wire-ios-request-strategy/sourcery/generated/AutoMockable.generated.swift"
```

Comment:
What is the idea to run SwiftLint on generated files? Anyway they are never programmed by a developer or do I miss something?

Reply:
Okay :)